src/modules/esl-share/README.md

```diff
@@ -306,43 +333,52 @@ The popup is notified of any configuration changes. In this case, the component
-
-There are two modes available to render buttons. In list mode, the `ESLShareList` is drawn inside the component as a list. Nothing special. 
-
-Similarly, in popup mode, the `ESLSharePopupTrigger`` is drawn inside component and shows a popup with the specified list of buttons.
-
-#### Public API
+#### Events
 
- - `config` - static method to get or update config with a promise of a new config object or using a config provider function (***Deprecated***)
+ - `esl:share:ready` - event to dispatch on ready of `ESLShare`
 
 ### ESLShareAction
 
 **ESLShareBaseAction** is an interface that describes share actions. Custom user's actions realizing this interface should have next public methods:
  - `isAvailable` - checks if this action is available on the user's device
  - `share` - performs an action to share
+
+### List of share buttons, the configuration of which we provide
+
+We provide configurations for share buttons for popular social networks and distribution methods. All you need to do to utilize this feature is to import a file with the configuration (see the section on `Configuring the ESLShare components`). All necessary activations and registrations will be carried out during the import.
+
+Here is the list of available buttons for sharing:
+
+ - `blogger`- shares data into Blogger an online content management system
+ - `copy` - copies the share data into a clipboard
+ - `facebook` - shares data into Facebook a free social networking website
+ - `hatena` - shares data into Hatena a japan free blogging service
+ - `kakao` - shares data into Kakao the most popular network among Koreans
+ - `line` - shares data into Line a popular messaging and social media platform in Japan and other parts of Asia
+ - `linkedin` - shares data into LinkedIn a social networking site designed specifically for the business community
+ - `mail` - activates the default mail client on the device for sending data via e-mail
+ - `mix` - shares data into Mix a social network service
+ - `mixi` - shares data into Mixi an online Japanese social networking service
+ - `myspace` - shares data into MySpace a social media platform
+ - `native-share` - activates the native sharing mechanism of the device to share data
+ - `pinterest` - shares data into Pinterest a social curation website for sharing and categorizing images found online
+ - `print` - prints page
+ - `pusha` - shares data into Pusha a social network service
+ - `reddit` - shares data into Reddit an American social news aggregation, content rating, and discussion website 
+ - `sina-weibo` - shares data into Sina Weibo a microblogging website and app which compares to Twitter and Instagram
+ - `telegram` - shares data into Telegram a cloud-based, cross-platform instant messaging (IM) service
+ - `twitter` - shares data into Twitter a free social networking site where users broadcast short posts known as tweets
+ - `wykop` - shares data into Wykop a Polish social networking internet service
```

Comment:
Just a cosmetic suggestion:
possibly it makes sense to split into two lists social media and special actions (copy, print, native, mail)

```suggestion
 - Social medias:
    - `blogger`- shares data into Blogger an online content management system
    - `facebook` - shares data into Facebook a free social networking website
    - `hatena` - shares data into Hatena a japan free blogging service
    - `kakao` - shares data into Kakao the most popular network among Koreans
    - `line` - shares data into Line a popular messaging and social media platform in Japan and other parts of Asia
    - `linkedin` - shares data into LinkedIn a social networking site designed specifically for the business community
    - `mix` - shares data into Mix a social network service
    - `mixi` - shares data into Mixi an online Japanese social networking service
    - `myspace` - shares data into MySpace a social media platform
    - `pinterest` - shares data into Pinterest a social curation website for sharing and categorizing images found online
    - `pusha` - shares data into Pusha a social network service
    - `reddit` - shares data into Reddit an American social news aggregation, content rating, and discussion website 
    - `sina-weibo` - shares data into Sina Weibo a microblogging website and app which compares to Twitter and Instagram
    - `telegram` - shares data into Telegram a cloud-based, cross-platform instant messaging (IM) service
    - `twitter` - shares data into Twitter a free social networking site where users broadcast short posts known as tweets
    - `wykop` - shares data into Wykop a Polish social networking internet service
 - Simple Actions
   - `copy` - copies the share data into a clipboard
   - `mail` - activates the default mail client on the device for sending data via e-mail
   - `native-share` - activates the native sharing mechanism of the device to share data
   - `print` - prints page
```